aii-core/src/main/perl/Shellfe.pm

```diff
@@ -30,8 +30,9 @@ use File::Path qw(mkpath rmtree);
 use File::Basename qw(basename);
 use DB_File;
 use Readonly;
-
+use Parallel::ForkManager 0.7.6;
```

Comment:
The ability for the parent to retrieve data structures is new as of version 0.7.6. We don't strictly need it, but useful for testing/debugging